dbsnp-importer/src/test/resources/application.properties

```diff
@@ -1,10 +1,25 @@
+dbsnp.datasource.driver-class-name=org.hsqldb.jdbcDriver
+dbsnp.datasource.url=jdbc:hsqldb:mem:db;sql.syntax_pgs=true;DB_CLOSE_DELAY=-1
+dbsnp.datasource.username=SA
+dbsnp.datasource.password=
+dbsnp.datasource.schema=src/test/resources/dbsnp-schema.sql
+dbsnp.datasource.data=src/test/resources/dbsnp-data.sql
+
+
+spring.jpa.hibernate.ddl-auto=none
+
 spring.datasource.driver-class-name=org.hsqldb.jdbcDriver
-spring.datasource.url=jdbc:hsqldb:mem:db;sql.syntax_pgs=true;DB_CLOSE_DELAY=-1
+spring.datasource.url=jdbc:hsqldb:mem:jobRepositoryDb;sql.syntax_pgs=true;DB_CLOSE_DELAY=-1
 spring.datasource.username=SA
 spring.datasource.password=
 
-# The same embedded database is reused by your whole test suite regardless of the number of application contexts.
-# Set to true to make sure that each context has a separate embedded database.
-spring.datasource.generate-unique-name=true
 
-spring.jpa.hibernate.ddl-auto=none
\ No newline at end of file
+processor=SubSnpCoreFieldsToEvaSubmittedVariantProcessor
+dbsnpBuild=150
```

Comment:
We are using dashes in other properties to try to adhere to the Spring style. This is something that the ConfigurationProperties [can handle](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-relaxed-binding), so let's be consistent.